packages/flutter/lib/src/widgets/text_selection.dart

```diff
@@ -110,11 +110,16 @@ class ToolbarItemsParentData extends ContainerBoxParentData<RenderBox> {
 ///
 /// Override text operations such as [handleCut] if needed.
 abstract class TextSelectionControls {
-  /// Builds a selection handle of the given type.
+  /// Builds a selection handle of the given `type`.
   ///
   /// The top left corner of this widget is positioned at the bottom of the
   /// selection position.
-  Widget buildHandle(BuildContext context, TextSelectionHandleType type, double textLineHeight);
+  ///
+  /// The supplied [onTap] should be invoked when the handle is tapped, if such
+  /// interaction is allowed. As a counterexample, the default selection handle
+  /// on iOS [cupertinoTextSelectionControls] does not call [onTap] at all,
+  /// since its handles are not meant to be tapped.
+  Widget buildHandle(BuildContext context, TextSelectionHandleType type, double textLineHeight, VoidCallback? onTap);
```

Comment:
@LongCatIsLooong This is a **breaking change** 😓 How are we supposed to deal with packages that should work with multiple versions of Flutter that call `buildHandle`.

https://pub.dev/packages/flutter_math_fork is such a package and now it cannot work.

The change should have been `[VoidCallback? onTap]`, otherwise it is breaking. Should I make a PR and we cherry pick a hot fix for the dev channel?

Reply:
So `beta` and `stable` have 3 positional parameters and `dev` has 4 positional parameters...

Reply:
Yeah the proposed change sounds good to me. Sorry for the breakage!

Per the [breaking change policy](https://github.com/flutter/flutter/wiki/Tree-hygiene#1-determine-if-your-change-is-a-breaking-change) this is not a breaking change so I'd recommend contributing tests to `flutter/tests` to prevent future breakage.
I'm not sure we have a process for cherry-picking a commit to the dev channel. See https://github.com/flutter/flutter/wiki/Flutter-Cherrypick-Process. Maybe we should add that dev build to https://github.com/flutter/flutter/wiki/Bad-Builds.